app/javascript/app/components/ghg-emissions/ghg-emissions-actions.js

```diff
@@ -23,10 +23,31 @@ const fetchGhgEmissionsMeta = createThunkAction(
           if (data) {
             const dataParsed = {};
             Object.keys(data).forEach(key => {
-              dataParsed[key] = data[key].map(item => ({
-                value: item.id,
-                label: upperFirst(item.name)
-              }));
+              dataParsed[key] = data[key].map(item => {
```

Comment:
So here is all the ugliness that we discussed this morning... Sorry that my changes didn't help that much.

Perhaps it might help the code look a bit better if the transformation is factored out into a separate function.